.azure-pipelines.yml

```diff
@@ -143,6 +147,32 @@ stages:
       iosCertificateFile: $(AppStoreCertificate)
       iosVariableGroup: 'ApplicationTemplate.Distribution.AppStore'
 
+- stage: Build_Canary
+  condition: and(eq(variables['IsCanary'], 'true'))
```

Comment:
We could add $[ne(variables['Build.Reason'], 'PullRequest')] to be sure not to build this on PR, if for example someone where to do a PR from a branch that starts with "canaries".

Reply:
Is that not an edge case, I don't see the use of PRs that starts with "canaries"